qa/sys/centos/bootstrap.sh

```diff
@@ -2,11 +2,3 @@
 
 yum update
 yum install -y java-1.8.0-openjdk-devel.x86_64
-
-##
-# Install logstash manually from a URL
-##
-BRANCH=${LOGSTASH_BRANCH:-'master'}
```

Comment:
Lets try to find a middle ground  here.

The problem with the downloading the file at provisioning time is this operation is done once unless we destroy the machine **OR** manually run the vagrant provision.
Are we at risk to run the tests not on the latest released version?

This problem doesn't appear on CI because on every run the machine are rebuild from scratch, but locally as human we wont be doing that since Its really time consuming.

I agree having the spec download the file could be slower, but nothing prevent us from persisting it. We can do a head check on it to see if we need to update it? This would fix the problem of the slow.

Another question, am I wrong or this file is only needed in the upgrade scenario? Sot this file is only needed to be downloaded in a `before :all` block? 

In a normal test run on CI, downloading from bootstrapping vs once in a before :all should yield the same execution time?